### PR TITLE
small fixes for postgres

### PR DESCRIPTION
**Pull request by demianfe**
I've been using ormin and I had some issues with postgres, this changes made it work.

### CODE REVIEW

File: ormin/ormin_postgre.nim [hunk 1]
```diff
@@ -93,11 +95,14 @@ template bindResult*(db: DbConn; s: PStmt; idx: int; dest: bool;
...
-proc fillString(dest: var string; src: cstring; srcLen: int) =
-  if dest.isNil: dest = newString(srcLen)
-  else: setLen(dest, srcLen)
-  copyMem(unsafeAddr(dest[0]), src, srcLen)
-  dest[srcLen] = '\0'
+proc fillString(dest: var string; src: cstring; srcLen: int) {.inline.} =
+  var
+    c = src[0]
+    i = 0
+  while c != '\0':
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: write that as 

```nim
var i = 0
while src[i] != '\0':
  dest.add src[i]
  inc i
```

File: ormin/ormin_postgre.nim [hunk 2]
```diff
@@ -25,13 +25,15 @@ proc c_strtod(buf: cstring, endptr: ptr cstring = nil): float64 {.
...
 
-var sid {.compileTime.}: int
+var sid = 0
+
+template genSid*: untyped =
+  inc(sid)
+  sid
 
 proc prepareStmt*(db: DbConn; q: string): PStmt =
-  static:
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: Seems worse than my solution. Why is it required?

**Comment by demianfe** [comment]
Done, apparently the problem was somewhere else.

File: ormin/ormin_postgre.nim (hunk 2 above)
Comment (demianfe) [reply ormin/ormin_postgre.nim]: actually this does not work, the constant name does not exists when the rest of the proc is executed giving this error message: 

ormin/ormin/ormin_postgre.nim(35, 20) Error: undeclared identifier: 'name'

**Pull request merged**